docs/sqllab.rst

```diff
@@ -112,11 +112,11 @@ of queries before executing this. Currently, Presto is supported in SQL Lab. To
 enable query cost estimation, add the following keys to the "Extra" field in the
 database configuration:
 
-.. code-block:: json
+.. code-block:: text
 
     {
         "version": "0.319",
-        "cost_estimate_enabled": true,
+        "cost_estimate_enabled": true
```

Comment:
Isn't `json` supported in the current version? According to this it should've been included for quite some time:

https://stackoverflow.com/questions/4259105/which-sphinx-code-block-language-to-use-for-json

Reply:
Yes it's supported, the problem is that by including `...` at the end, we are making JSON invalid, and sphinx is going to validate that. So it's a question of remving `...` at the end (and that would give us proper JSON highlight) or change the type to `text`

Reply:
Right, didn't think of that. Probably better to stick with `text` 👍